qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/1.1.send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: 1.1.Send CV
+
+  Scenario: 1.1_01 positive sending CV
+  Given user opened the form "Send CV"
+  When user enters valid data in the field "Name"
+  And user enters valid data in the field "Email"
+  And user attaches CV
+  And user marks "I agree to the personal data processing"
```

Comment:
We can write it like this: "user marks checkbox with agreement to the personal data processing", I guess

Reply:
done